app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeFragment.kt

```diff
@@ -260,9 +258,13 @@ class AttendeeFragment : Fragment(), ComplexBackPressFragment {
             .observe(viewLifecycleOwner, Observer {
                 loadEventDetailsUI(it)
                 setupPaymentOptions(it)
-                if (attendeeViewModel.pendingOrder.value != null) {
-                    setupCountDownTimer(it)
-                }
+            })
+
+        attendeeViewModel.getSettings()
```

Comment:
There is no guarantee that settings have been fetched till the next statement is executed

Reply:
Then onErrorResumeNext will execute and settings will fetch from api.

Reply:
Still there is no guarantee that settings have been fetched till the next statement is executed, as getsettings is asynchronous

Reply:
Sorry, I don't understand which next statement. During fetching/getting settings progress dialog is displaying and if settings fetch successfully or there is an error(=15), value for mutableOrderExpiryTime is set and only then countdown timer will start.

Reply:
The statement which is immediately proceeding getSettings